<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnswerStatistics</class>
 <widget class="QWidget" name="AnswerStatistics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>60</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>60</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>60</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>240</width>
    <height>60</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AnswerStatistics</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#AnswerStatistics{
background-color: rgb(255, 255, 255);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <widget class="QLabel" name="label_9">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_leftFill">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-bottom:1px solid #C4C4C4;
border-top:1px solid #C4C4C4;
border-left:1px solid #C4C4C4;
border-top-left-radius : 20px;  
border-bottom-left-radius : 20px; 


</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_progress" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widget_progress{
border-bottom:1px solid #C4C4C4;
border-top:1px solid #C4C4C4;
background-color: rgb(255, 255, 255,0);
font: 10pt &quot;微软雅黑&quot;;
color: rgb(68, 68, 68);
}
</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>50</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>162</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">
border-bottom:1px solid #C4C4C4;
border-top:1px solid #C4C4C4;
background-color: rgb(255, 255, 255,0);
font: 10pt &quot;微软雅黑&quot;;
color: rgb(68, 68, 68);</string>
        </property>
        <property name="text">
         <string>A</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>140</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_stat">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-bottom:1px solid #C4C4C4;
border-top:1px solid #C4C4C4;
background-color: rgb(255, 255, 255,0);
font: 10pt &quot;微软雅黑&quot;;
color: rgb(119, 119, 119);</string>
        </property>
        <property name="text">
         <string>0%(0人)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_rightFill">
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-bottom:1px solid #C4C4C4;
border-top:1px solid #C4C4C4;
border-right:1px solid #C4C4C4;
border-top-right-radius : 20px;  
border-bottom-right-radius : 20px; 



</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_10">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
